### PR TITLE
Feature spoke entity service

### PR DESCRIPTION
**Pull request by gurpreetbml**

### CODE REVIEW

File: utilities/path_utilities.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+            ql="select * where channel_id={}".format(path_id)
+        )
+        if len(channel_history):
+            ug.delete_entity(
+                "/channelconfighistories/{}".format(channel_history[0]['uuid'])
+            )
+
+    def get_user_paths(self, user_id, doc_type, is_show=False, order=None, cursor=None):
+        path_count = 10
+        curator_path_cursor = ''
```
Comment (nojyarg) [inline utilities/path_utilities.py]: why do we init these cursors as empty strings and not None?

File: utilities/path_utilities.py [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+            '/organizations/{}'.format(path['org_id'])
+        )
+        if organization and 'title' in organization:
+            org_data = {
+                'name': organization['title'],
+                'uuid': organization['uuid']
+            }
+            path['organization'] = org_data
+
+    def remove_path(self, path):
```
Comment (nojyarg) [inline utilities/path_utilities.py]: just a general though...deleting entities in ug should happen asynchronously, if at all possible.

File: utilities/path_utilities.py [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+
+from utilities.image_utilites import *
+from utilities.rec_engine_utilities import *
+from utilities.channel_feed_utilities import *
+from utilities.search_service_utilities import SearchUtilities
+from config_loader import ug
+from models.path import Path
+from utilities.model_utilities import *
+
+class Path_utilities:
```
Comment (nojyarg) [inline utilities/path_utilities.py]: lowercase class name to match .py file? or PathUtilities?

File: utilities/homefeed_utilities.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+import json
+
+from utilities.sqs_utilities import *
+
+def notify_homefeed_service(action, entity_id, key_id, user_id=None):
+    home_feed_msg = {
```
Comment (nojyarg) [inline utilities/homefeed_utilities.py]: sometimes this message would contain more than one attribute in the data block (see homefeed-service /events/event_router.py) - let's sync on this later today or tomorrow.

File: utilities/ug_utilities.py [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+    def upload_file_to_usergrid(self, file, ug_entity):
+        file_name = secure_filename(file.filename)
+        out_file = '{0}/{1}'.format(
+            tempfile.gettempdir(),
+            file_name
+        )
+        file.save(out_file)
+        audiojson = {
+            "filename": file_name,
+            "name": "{}{}".format(
```
[the remark this answers is not in the document]
Comment (gurpreetbml) [reply utilities/ug_utilities.py]: will change that in code

File: utilities/path_utilities.py (hunk 1 above)
Comment (gurpreetbml) [reply utilities/path_utilities.py]: I will change that

File: utilities/path_utilities.py (hunk 2 above)
Comment (gurpreetbml) [reply utilities/path_utilities.py]: will be done when I will add celery tasks

File: utilities/path_utilities.py (hunk 3 above)
Comment (gurpreetbml) [reply utilities/path_utilities.py]: will do that

**Pull request merged**